internal/controller/standalone_pgadmin/pod.go

```diff
@@ -342,3 +343,31 @@ if os.path.isfile('` + ldapPasswordAbsolutePath + `'):
 
 	return append([]string{"bash", "-ceu", "--", script, "startup"}, args...)
 }
+
+// PodSecurityContext returns a v1.PodSecurityContext for pgadmin that can write
+// to PersistentVolumes.
+func PodSecurityContext(r *PGAdminReconciler, cluster *v1beta1.PGAdmin) *corev1.PodSecurityContext {
+	podSecurityContext := initialize.PodSecurityContext()
+
+	// // Use the specified supplementary groups except for root. The CRD has
+	// // similar validation, but we should never emit a PodSpec with that group.
+	// // - https://docs.k8s.io/concepts/security/pod-security-standards/
+	// for i := range cluster.Spec.SupplementalGroups {
+	// 	if gid := cluster.Spec.SupplementalGroups[i]; gid > 0 {
+	// 		podSecurityContext.SupplementalGroups =
+	// 			append(podSecurityContext.SupplementalGroups, gid)
+	// 	}
+	// }
+
+	// OpenShift assigns a filesystem group based on a SecurityContextConstraint.
+	// Otherwise, set a filesystem group so PostgreSQL can write to files
+	// regardless of the UID or GID of a container.
+	// - https://cloud.redhat.com/blog/a-guide-to-openshift-and-uids
+	// - https://docs.k8s.io/tasks/configure-pod-container/security-context/
+	// - https://docs.openshift.com/container-platform/4.8/authentication/managing-security-context-constraints.html
```

Comment:
(Are these links still valid? I didn't get a chance to check this afternoon.)

Reply:
The links WFM. This last one could be updated to a more recent version number.

https://docs.openshift.com/container-platform/4.14/authentication/managing-security-context-constraints.html

Reply:
Yep, I was going to bump that OCP version.